ocaml/forkexecd/test/fe_test.ml

```diff
@@ -140,6 +140,10 @@ let fail x =
   Printf.fprintf stderr "%s\n" x ;
   assert false
 
+let check_string s expected =
```

Comment:
Maybe call this `expect` and switch argument order? The existing code in this module is a bit clumsy and today we would probably use a test framework.

Reply:
done